ADAL/src/request/ADAuthenticationRequest+AcquireToken.h

```diff
@@ -32,7 +32,8 @@
 
 // Bypasses the cache and attempts to request a token from the server, generally called after
 // attempts to use cached tokens failed
-- (void)requestToken:(ADAuthenticationCallback)completionBlock;
+- (void)requestToken:(ADAuthenticationError*)previousError
```

Comment:
Try to minimize parameters on methods unless there's no way around it, otherwise the number of parameters has a tendency to explode. You should use ivars instead when you're in ADAuthenticationRequest